cli/test/e2e/package_authoring_e2e_test.go

```diff
@@ -1289,3 +1290,188 @@ spec:
 			RunOpts{StdinReader: promptOutput.StringReader(), StdoutWriter: promptOutput.BufferedOutputWriter()})
 	})
 }
+
+func TestE2EInitAndReleaseByAddingTag(t *testing.T) {
+	input := E2EAuthoringTestCase{
```

Comment:
It seems that you are trying to use table tests but I don't see that in the later section of the test 🤔 , also the other test `TestE2EInitAndReleaseByAddingInvalidTag` seems like it should just be another variant of this test only.

Reply:
No, I am not trying to use a table test, But I am reusing the same struct which is being used in table test as that struct binds up everything. Also, `TestE2EInitAndReleaseByAddingInvalidTag` throws out an error during release while this test does release and then install the package. IMO these are two different tests.

Reply:
> But I am reusing the same struct which is being used in table test as that struct binds up everything

I think we shouldn't use it if we are not planning to add table tests, it's just adding an extra layer which doesn't seem to be useful in the test.

> Also, TestE2EInitAndReleaseByAddingInvalidTag throws out an error during release while this test does release and then install the package.

Yeah, if we are not using table tests, then I think a separate one makes sense.

nit:
`TestE2EInitAndReleaseByAddingTag`  -> `TestPackageInitAndReleaseWithTag`
`TestE2EInitAndReleaseByAddingInvalidTag` -> `TestPackageRepoInitAndReleaseWithInvalidTag`

Reply:
Adopted

Reply:
> I think we shouldn't use it if we are not planning to add table tests

@rohitagg2020 What do you think about this?